handlers/tokeninfo/jwt/handler.go

```diff
@@ -0,0 +1,63 @@
+package jwthandler
+
+import (
+	"encoding/json"
+	"github.com/dgrijalva/jwt-go"
+	"net/http"
+	"strings"
+	"github.com/zalando/planb-tokeninfo/keys"
+	"github.com/zalando/planb-tokeninfo/handlers/tokeninfo"
+	"github.com/coreos/dex/pkg/log"
+)
+
+type jwtHandler struct {
+	keyLoader keys.KeyLoader
+}
+
+func (h *jwtHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	ti, err := h.validateToken(r)
+	if err != nil {
+		// TODO: consider a debug mode or log
+		// as we are no longer returning error details to the user
+		switch err {
+		case jwt.ErrNoTokenInRequest:
+			tokeninfo.Error(w, tokeninfo.ErrInvalidRequest)
+		default:
+			log.Debug(err)
+			tokeninfo.Error(w, tokeninfo.ErrInvalidToken)
+		}
+		return
+	}
+
+	w.Header().Set("Content-Type", "application/json")
+	w.WriteHeader(http.StatusOK)
+	json.NewEncoder(w).Encode(ti)
+}
+
+func (h *jwtHandler) Match(r *http.Request) bool {
+	token := r.URL.Query().Get("access_token")
```

Comment:
that's not enough, we need to support the `Authorization` header too, see https://github.com/zalando/planb-tokeninfo/commit/204e94502df4738e8a981d556ef0b3fb5e62955a